source/Halibut/Transport/SecureWebSocketListener.cs

```diff
@@ -114,7 +115,7 @@ async Task Accept()
                     {
                         if (!cts.IsCancellationRequested)
                         {
-                            log.WriteException(EventType.ErrorInInitialisation, "Error accepting Web Socket client", ex);
+                            log.WriteException(EventType.ErrorInInitialisation, "Error accepting Web Socket client: {0}", ex, context?.Request.RemoteEndPoint);
```

Comment:
Should these use the extension method or is it a different APi that doesn't throw

Reply:
The latter - it doesn't throw but I just checked and it _can_ be null. I'm not sure what the log method will do with a null object, so I'll check that behaviour and update the code if necessary.